### DISCUSSION TITLE
Trix Toolbar Nesting (indent) buttons

### DISCUSSION THREAD

**Discussion by Earn717** [filamentphp/filament#6136] [Help]
Is there a simple way to enable the Trix toolbar's indent buttons?   I've tried variations of enableToolbarButtons to no avail.

`->enableToolbarButtons(['increaseNestingLevel'])`

`<button type="button" class="trix-button trix-button--icon trix-button--icon-decrease-nesting-level" data-trix-action="decreaseNestingLevel" title="' + t3.outdent + '" tabindex="-1">' + t3.outdent + '</button>`

`<button type="button" class="trix-button trix-button--icon trix-button--icon-increase-nesting-level" data-trix-action="increaseNestingLevel" title="' + t3.indent + '" tabindex="-1">' + t3.indent + '</button>`

**Comment by danharrin** [comment]
You'd need to send a PR to add this to our editor
